CMakeLists.txt

```diff
@@ -482,9 +482,6 @@ if(SENTRY_BACKEND_CRASHPAD)
 		endif()
 	endif()
 	add_dependencies(sentry crashpad::handler)
-	if(CRASHPAD_WER_ENABLED)
-		add_dependencies(sentry crashpad::wer)
```

Comment:
You can move `add_compile_definitions(CRASHPAD_WER_ENABLED)` here and move `add_dependencies(sentry crashpad::wer)` right after the `add_subdirectory()` invocation in the `false` branch of  `if(SENTRY_CRASHPAD_SYSTEM)`.